### DISCUSSION TITLE
網頁下方的介紹區可以放什麼

### DISCUSSION THREAD

**Discussion by gasolin** [lun-dao/LunDAO#33] [General]
目前的版本

<img width="1119" alt="截圖 2022-01-06 上午11 14 00" src="https://user-images.githubusercontent.com/748808/148322590-d4364e68-55fd-4bd2-a355-14d112cfcf08.png">

下方有三塊 placeholder 尚未有內容，
但直接拿掉的話，會有一大塊空白不美觀。
要放什麼哩？

較詳細的說明, 推薦文章, tweet embed？

**Comment by gasolin** [comment]
<img width="1085" alt="截圖 2022-01-06 下午5 27 31" src="https://user-images.githubusercontent.com/748808/148360798-185fa3cb-3681-432d-ae01-8044d607ba84.png">

一種用法是在這邊列出對不同類人群提供的簡介和 action button

讀者 -> Blog
寫作者 -> Github
捐獻者 -> ?

或參考 showcast 看別的站怎麼擺
https://docusaurus.io/showcase

**Comment by yurenju** [comment]
覺得可以擺在幾個重點：
- 撰寫：可以說明我們有在募集文章，方式是怎樣
- 貢獻：說明專案是社群性質，社群經營的部分是大家一起完成，可以參考 CONTRIBUTING.md 進行貢獻
- 閱讀：導引至文章處瀏覽

我覺得這三點算是 LunDAO 的 call to action。看看大家還有沒有覺得更重要的事情應該放在這三欄。
